wp-dependency-installer.php

```diff
@@ -144,12 +149,12 @@ public function register( $config, $plugin_path ) {
 			$source = basename( $plugin_path );
 			foreach ( $config as $dependency ) {
 				$dependency['source']    = $source;
+				$dependency['sources'][] = $source;
 				$slug                    = $dependency['slug'];
-				$dependency['tooltip'][] = $source;
+				if ( isset( $this->config[ $slug ] ) && $this->is_required( $dependency ) ) {
+					$dependency['sources'] = array_merge( $this->config[ $slug ]['sources'], $dependency['sources'] );
+				}
 				if ( ! isset( $this->config[ $slug ] ) || $this->is_required( $dependency ) ) {
```

Comment:
This is not directly related to this pull, but why we should also check that the dependency currently `is required`?

Reply:
From what I understood in the case of a collision of dependencies (shared dependency) it keeps in memory only the last registered configuration.

Anyway, it can be a good thing document this behavior within the wiki.